### ISSUE TITLE
How to build a package like gtk-d with just subPackages

### ISSUE THREAD

**Issue by timotheecour** [dlang/dub#1039]
```
cd ~/.dub/packages/gtk-d-3.4.1/gtk-d
dub build
```
Main package must have a binary target type, not none. Cannot build.

This also doesn't work (there's not configurations, just subPackages)
```
dub build --config=gtkd
```

=> how do I build this?

**Comment by BBasile** [comment]
I've already encountered this case because it also happens when `targetType` is set to `sourceLibrary`. The responsible is clearly gtkd. They should set the target to `staticLibrary`, otherwise you can't build it but only add it as dependency for each application that uses gtkD.

**Comment by timotheecour** [comment]
found the answer:
```dub build gtk-d:gtkd```

where is that documented?
not seeing this in `dub build --help` for example

**Comment by timotheecour** [comment]
reply from Basile Burg (he replied via email; not sure why it doesn't appear here):

> I've already encountered this case because it also happens when targetType is set to sourceLibrary. The responsible is clearly gtkd. They should set the target to staticLibrary, otherwise you can't build it but only add it as dependency for each application that uses gtkD.

**Comment by timotheecour** [comment]
Basile, not sure why your reply doesn't appear on the github issue

**Comment by WebFreak001** [comment]
duplicate of #97

fixed in #1364
